### PR TITLE
Clarify output directory/`--dir` parameter for monitor service.

### LINKED ISSUE #96: need instructions on troubleshooting installation.
on a MacBook Air, intel hw, OS Monterey, I was able to install the node.js modules (via sudo) and start the service, but cannot view the output of a local run.

bash console:

```
~/github/zmorris/2023_stancon_workflow/runs (main)> npx mcmc-monitor@latest start --dir ./air_simple --verbose

Connect on local machine: https://flatironinstitute.github.io/mcmc-monitor?s=http://localhost:61542

Server is running on port 61542
 getRuns
 getRuns
 getRuns
```

- each `getRuns` is triggered by web browser request
```
https://flatironinstitute.github.io/mcmc-monitor/?s=http://localhost:61542
```

this is per the readme.

it would be helpful to have an example of how to monitor the output of a local run.

also, signing up for the Heroku app is quite involved - you need to install the Salesforce 2FA app (Authy didn't seem to work here) just to access your account.  as I don't understand much about this proxy service, I didn't pursue this option.   again, it would be useful to explain why one would want to do this instead of letting Flatiron run the proxy.

update:  I realized my error - I needed to specify "--dir .", the parent directory of my output runs.   the monitor is beautiful.  more information on where the output directories are, also explain that in CmdStanPy and CmdStanR it is necessary to specify arg "output_dir" to the sample command (because tmp dirs are hard to find).

### PR DESCRIPTION
**Pull request by jsoules**
Fix #96 

Hi @mitzimorris -- would this revision to the doc be sufficient, or is there more we should add? Thanks!

### CODE REVIEW

**Review by mitzimorris — APPROVED** [review]
nice and concise!

**Comment by jsoules** [comment]
@mitzimorris Thanks! I actually just expanded it a little bit in response to your comments from the issue--mind taking a look? Trying to balance keeping it concise with providing a concrete example...

File: README.md [hunk 1]
```diff
@@ -54,14 +54,28 @@ npm install -g @mapbox/node-pre-gyp
...
+
+For example, suppose we run `mcmc-monitor` with `--dir /home/user/examples/`, and that this directory contains two
+subdirectories, `multi-normal-1` and `multi-normal-2`, each containing the output of a Stan run (completed or
+still in progress). Then the monitor home page will list two runs, linked as `multi-normal-1` and `multi-normal-2`.
+If another process subsequently writes an analysis to `/home/user/examples/analysis-3`, then the monitor will
+begin displaying a run called `analysis-3` as well.
+
+Note that there is not currently a way to change the monitored directory while the monitor is running.
+Some tools (such as `CmdStanPy` and `CmdStanR`) use `tmp` directories by default; in this case, you will need
+to provide an `output_dir` argument for those packages which should match the path given to `mcmc-monitor`.
```
Comment (mitzimorris) [inline README.md]: you have 2 choices:  make the monitor use the tmp directory or make the CmdStanPy/R interface send outputs to specified subdirectories under the monitored directory.

Comment (jsoules) [reply README.md]: How is:

> Note that there is not currently a way to change the monitored directory while the monitor is running.
Some tools (such as `CmdStanPy` and `CmdStanR`) use `tmp` directories by default; to monitor the output from
these programs, you must either tell MCMC Monitor to monitor the `tmp` directory, or use an `output_dir` argument
to send program output to specified subdirectories under the monitored directory.

?

Comment (mitzimorris) [reply README.md]: unfortunately, it seems that CmdStanPy and CmdStanR behave differently w/r/t creating output directories.
CmdStanPy will create the output directory as needed; CmdStanR won't - it throws an error - I'll file an issue there.

I think that CmdStanR/CmdStanPy details should be discussed in detail elsewhere, with examples.

Comment (mitzimorris) [reply README.md]: I think that revised version is good.

Comment (jsoules) [reply README.md]: Great, I'll push this (and defer discussion of the `CmdStan`s to elsewhere).

Thanks for raising this & helping to hash it out!

**Pull request merged**